test/browser/component-utils/controlled-app.js

```diff
@@ -1,32 +1,33 @@
 /* eslint-env mocha */
 /* global expect */
 
+import requestAnimationFrameCB from 'raf';
+
+const raf = () => new Promise(requestAnimationFrameCB);
```

Comment:
Rather than repeating this boilerplate in each browser test module, how about pulling it into a shared test util?
```js
import {nextAnimationFrame} from '../../util';
```
(FWIW I also prefer the existing usage of `await nextAnimationFrame()` to `await raf()`)

Reply:
Sure